shell/platform/darwin/ios/framework/Headers/FlutterEngine.h

```diff
@@ -119,6 +123,15 @@ FLUTTER_EXPORT
  */
 - (BOOL)runWithEntrypoint:(NSString*)entrypoint libraryURI:(NSString*)uri;
 
+/**
+ * Destroy running context for an engine.
+ *
+ * If you create a FlutterEngine with allowHeadlessExecution set to YES,
+ * call destroyContext explicitly when you want to release the engine.
```

Comment:
I would change these two lines to say:

```
This method can be used to force the FlutterEngine object to release all resources. After sending this message, the object will be in an unuseable state until it is deallocated. Accessing properties or sending messages to it will result in undefined behavior or runtime errors.
```